nuxeo-drive-client/nxdrive/data/ui5/js/ndrive-conflicts.js

```diff
@@ -54,16 +54,27 @@ ConflictsController.prototype.updateErrors = function($scope) {
 	$scope.ignoreds = ignoreds;
 }
 ConflictsController.prototype.updateConflicts = function($scope, $interval) {
+    var lastContributorPending = false;
 	self = this;
 	$scope.conflicts = angular.fromJson(drive.get_conflicts());
 	for (var i=0; i<$scope.conflicts.length; i++) {
+		lastContributorPending = lastContributorPending || $scope.conflicts[i].need_refresh;
 		if ($scope.conflicts[i].last_error == "DUPLICATING") {
 			setTimeout( function() {
 				self.updateConflicts($scope, $interval);
 				$scope.$apply();
 			}, 1000);
 		}
 	}
+	if (lastContributorPending) {
```

Comment:
The delay is required only if the database does not have the userid value yet. (conflict happens after users table is retrieved for the day). If this scenario is solved in a different way (like as soon as conflict is detected) then we can avoid this delay and retrieve again.